tests/manage/z_cluster/cluster_expansion/test_add_capacity.py

```diff
@@ -5,8 +5,12 @@
 from ocs_ci.ocs.ocp import OCP
 from ocs_ci.ocs.resources import storage_cluster
 from ocs_ci.utility.utils import ceph_health_check
+import pytest
```

Comment:
Actually import order should be like here:
https://github.com/red-hat-storage/ocs-ci/pull/2022/files#diff-af67ca71910cab9689be42cb2305a33f

I covered this in my temp PR for lib bucket testing so probably you can close this one @keesturam

Reply:
got it. I'll close this PR.